python/lsst/obs/base/_instrument.py

```diff
@@ -398,6 +398,24 @@ def makeDataIdTranslatorFactory(self) -> TranslatorFactory:
         """
         raise NotImplementedError("Must be implemented by derived classes.")
 
+    @classmethod
+    def constructCollectionName(cls, label: str) -> str:
```

Comment:
How about just `make` instead of `construct`?  I feel like that's kind of the default verb for something like this unless there's a good reason to pick a different one.